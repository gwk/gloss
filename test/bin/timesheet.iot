{
  'cmd': ['$PROJ/bin/timesheet.py', 'test/bin/timesheet-test.txt'],
  'links': {'test':'test'}
}