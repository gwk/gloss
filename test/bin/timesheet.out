2014-10-30                                                      
14:00 intro, meeting                                            | 840 
16:00 end = 2:00                                                | 960 =  120m

TOTAL HOURS:    2:00
TOTAL EXPENSE: $0.00
TOTAL PAYMENT: $0.00
TOTAL:         $0.00
